cli/common/utils.go

```diff
@@ -34,6 +34,21 @@ type PackDependency struct {
 
 type PackDependencies []PackDependency
 
+type InstanceYaml struct {
```

Comment:
Seems that these params aren't a part of this PR. Let's process only `fd-limit` and `stateboard-fd-limit`. Moreover, I think that it would be easier to use this as a `string[interface{}]` map, we will discuss it on #562.